scripts/stats.py

```diff
@@ -0,0 +1,17 @@
+import random

```

Comment:
I think we should think about moving these into appropriate classes as we get more scripts

Reply:
I'm Invisioning we will eventually have factories and stuff, maybe.

Reply:
I agree. Ideally there will be a character class that generates the appropriate properties, and has methods for attacking. I have chosen not to do this until it is needed.